### PR TITLE
Table Expand Column Title Customization

### PR DESCRIPTION
**Pull request by losgif**
Table Expand Column Title Customization
表格配置增加一个自定义columnTitle属性

ref: #853

### CODE REVIEW

File: src/interface.ts [hunk 1]
```diff
@@ -200,6 +200,7 @@ export interface ExpandableConfig<RecordType> {
   expandedRowKeys?: readonly Key[];
   defaultExpandedRowKeys?: readonly Key[];
   expandedRowRender?: ExpandedRowRender<RecordType>;
+  expandedColumnTitle?: React.ReactNode;
```
Comment (afc163) [inline src/interface.ts:203]:
```suggestion
  columnTitle?: React.ReactNode;
```

Comment (losgif) [reply src/interface.ts:203]: ok, 其他地方也统一名称吗？

Comment (afc163) [reply src/interface.ts:203]: 统一，用户是通过 `expandable={{ columnTitle: 'xxx' }}` 来设置。

Comment (losgif) [reply src/interface.ts]: OK了，测试用例也写了一个。帮忙看看有啥缺的不

File: docs/examples/expandedRowRender.tsx [hunk 2]
```diff
@@ -56,7 +57,7 @@ const Demo = () => {
     { title: 'title 1', dataIndex: 'a', key: 'a', width: 100 },
     { title: 'title 2', dataIndex: 'b', key: 'b', width: 100 },
     { title: 'title 3', dataIndex: 'c', key: 'c', width: 200 },
-    { title: 'Operation', dataIndex: '', key: 'x', render: renderAction },
+    { title: 'Operation', dataIndex: '', key: 'x', render: renderAction }
```
Comment (afc163) [inline docs/examples/expandedRowRender.tsx:60]: Don't update these lines.

File: docs/examples/expandedRowRender.tsx [hunk 3]
```diff
@@ -15,24 +15,25 @@ interface RecordType {
...
 for (let i = 0; i < 10; i += 1) {
-  const str = `${i}`;
+  const str = `${ i }`;
   const item: RecordType = {
     key: i * 10 + 99,
     a: str.repeat(3),
     b: str.repeat(5),
     c: str.repeat(7),
-    d: i,
+    d: i
```
Comment (afc163) [inline docs/examples/expandedRowRender.tsx:28]: Don't update these lines.

File: docs/examples/expandedRowRender.tsx [hunk 4]
```diff
@@ -15,24 +15,25 @@ interface RecordType {
 const tableData: RecordType[] = [
   { key: 0, a: '123' },
   { key: 1, a: 'cdd', b: 'edd' },
-  { key: 2, a: '1333', c: 'eee', d: 2 },
+  { key: 2, a: '1333', c: 'eee', d: 2 }
```
Comment (afc163) [inline docs/examples/expandedRowRender.tsx:18]: Don't update these lines.

File: docs/examples/expandedRowRender.tsx [hunk 5]
```diff
@@ -15,24 +15,25 @@ interface RecordType {
...
 ];
 
 for (let i = 0; i < 10; i += 1) {
-  const str = `${i}`;
+  const str = `${ i }`;
```
Comment (afc163) [inline docs/examples/expandedRowRender.tsx:22]: Don't update these lines.

File: docs/examples/expandedRowRender.tsx [hunk 6]
```diff
@@ -87,14 +88,18 @@ const Demo = () => {
   if (expandedRowKeys.length) {
     const closeAll = () => setExpandedRowKeys([]);
     toggleButton = (
-      <button type="button" onClick={closeAll}>
+      <button type="button" onClick={ closeAll }>
```
Comment (afc163) [inline docs/examples/expandedRowRender.tsx:91]: Don't update these lines.

Comment (losgif) [reply docs/examples/expandedRowRender.tsx]: The code style in this file has been restored.

**Comment by afc163** [comment]
https://github.com/react-component/table/releases/tag/v7.26.0

**Pull request merged**